models/group.js

```diff
@@ -3,9 +3,15 @@ const mongoose = require('mongoose');
 const { Schema } = mongoose;
 
 const groupSchema = new Schema({
-    title: String,
-    description: String,
-    chat: { type: Schema.Types.ObjectId, ref: 'Chat' },
+  title: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    required: false,
+  },
+  chat: { type: Schema.Types.ObjectId, ref: 'Chat' },
```

Comment:
add required, since an inherited class, must reference to its super class Chat (existence condition)